One Master script to rule them all.

C-Pablo-Diego-Rosell opened the pull request:
We will use this master R script for the whole Cycle 1 analytics pipeline. The Master will: 
1. Set the working directory ("setwd") for the entire workflow. 
2. Run all package installs.
3. Source all sub-scripts.

C-Pablo-Diego-Rosell committed:
One Master script to rule them all. 

We will use this master R script for the whole Cycle 1 analytics pipeline. The Master will: 
1. Set the working directory ("setwd") for the entire workflow. 
2. Run all package installs.
3. Source all sub-scripts.

mhoover commented on NGS2_WITNESS_Cycle1_MASTER.R:
```diff
@@ -0,0 +1,59 @@
+
+### Response funnel 
+
+source ("NGS2_WITNESS_Cycle1_response.R") ### TO BE ADDED (Matt)
```
These `### TO BE ADDED` comments do not exist yet, correct? I would suggest that we at least comment these lines that don't exist yet to ensure a clean run if someone was to execute this. In addition, authors should be aware of the script names so we don't have a conflict down the road (i.e., I call the script something different than what is here right now).

mhoover commented:
This is pretty straightforward, but would suggest action on my comment above. Fixing that up and this gets a 👍 from me.

uduwage commented on NGS2_WITNESS_Cycle1_MASTER.R:
```diff
@@ -0,0 +1,59 @@
+
+### Set base directory
+
+setwd(X:/DARPA_NGS2/CONSULTING/Analytics/Cycle1)
```
We should try to set this to work on multiple example.
```
a = Sys.info()[1]
if( a == "Windows") { setwd("SET THE WINDOWS PATH")}
if( a != "Windows") { setwd("SET THE WINDOWS PATH")}
```

C-Pablo-Diego-Rosell replied:
Thanks Anu! I would like to make sure that we are working off of the X drive though.

mhoover replied:
Pablo, the issue is for Mac users, there is no X drive. So, the equivalent path for Anu and I, for example, is `/Volumes/Clients/DARPA_NGS2/CONSULTING/Analytics/Cycle1`. So, to Anu's point, the following should be good:

```
a = Sys.info()[1]
if( a == "Windows") { setwd("X:/DARPA_NGS2/CONSULTING/Analytics/Cycle1")}
if( a != "Windows") { setwd("/Volumes/Clients/DARPA_NGS2/CONSULTING/Analytics/Cycle1")}
```
That way, if Anu or I were to try and execute anything we wouldn't get an error.

C-Pablo-Diego-Rosell replied:
Thanks for clarifying Matt, will proceed with this change!

C-Pablo-Diego-Rosell committed:
Update NGS2_WITNESS_Cycle1_MASTER.R

Commented out the scripts that are pending per Matt's comment. I did want the script to fail at that point to ensure that we go back and get everything in place before running the full master, but I guess that's a bit silly!

C-Pablo-Diego-Rosell committed:
Bayesian analytics for experiment 1

Bayesian analytic script to complement frequentist analytical output: 
- Computations are done through the "rstanarm" package. 
- A custom function produces key Bayesian outputs. 
- Each hypothesis is run using the "rstanarm" weakly informative priors from the default setting (e.g. 'glm.out.4.1.1.default'), and strongly informative priors (logistic coefficient, standard errors) from the Rand experiment ((e.g. 'glm.out.4.1.1.inform').

C-Pablo-Diego-Rosell committed:
Delete NGS2_WITNESS_Cycle1_bayesian_exp1.R

uduwage reviewed (commented):
Its a good idea that we have a main file to call all the scripts. Only one comment on setwd.

C-Pablo-Diego-Rosell committed:
Update NGS2_WITNESS_Cycle1_MASTER.R

Implemented tweaks to setwd as requested by Anu/Matt
